content/en/docs/concepts/containers/container-lifecycle-hooks.md

```diff
@@ -50,7 +50,7 @@ A more detailed description of the termination behavior can be found in
 ### Hook handler implementations
 
 Containers can access a hook by implementing and registering a handler for that hook.
-There are two types of hook handlers that can be implemented for Containers:
+There are threetypes of hook handlers that can be implemented for Containers:
```

Comment:
Thanks for the fix! Just a minor typo in the proposed solution; please check the suggestion below.

```suggestion
There are three types of hook handlers that can be implemented for Containers:
```

Reply:
Oops my mistake! Thanks for review.